source/parser.hera

```diff
@@ -906,6 +917,8 @@ NestedBindingProperties
   # NOTE: Add NestedBindingProperties after rest from CoffeeScript
   PushIndent NestedBindingProperty*:props NestedBindingRestProperty?:rest NestedBindingProperty*:after PopIndent ->
     if (!(props.length || rest)) return $skip
+    props = props.flat()
+    after = after.flat()
```

Comment:
I think each `NestedBindingProperty*` should be `NestedPropertyList?`, given that there's no longer a delimiter in `BindingProperty`? (and omit `.flat()`)